component/src/main/java/io/siddhi/extension/io/cdc/source/metrics/CDCStatus.java

```diff
@@ -0,0 +1,9 @@
+package io.siddhi.extension.io.cdc.source.metrics;
```

Comment:
Add the license header.

Reply:
Hi @deshankoswatte, This issue was fixed in e46325f81ffb02df800c3d47edd17907babdc83f commit